tests/causal_refuters/base.py

```diff
@@ -50,6 +48,8 @@ def null_refutation_test(self, data=None, dataset="linear", beta=10,
             confounders_effect_on_outcome = self.confounders_effect_on_y,
             effect_strength_on_treatment =self.effect_strength_on_t,
             effect_strength_on_outcome=self.effect_strength_on_y)
+        print(ref)
```

Comment:
Better to remove such print statements, or switch them to a logger.debug message

Reply:
Added the logging function to the next commit.